1806 task station diamond make configurable

AKissmehl opened the pull request:
Diamond is now configurable. Please take a look at the changes in Open XR Package Setting.

fwild committed:
Update CalibrationTests.cs

increased time out to 15sec for the helper function EnsureTestReadiness (for loading work place model, etc.)

fwild committed:
Update CalibrationTests.cs

increased timeout to 60s

fominykh committed:
Update CHANGELOG.md

fwild committed:
Update linux_develop_workflow.yml

fwild committed:
Update linux_develop_workflow.yml

fwild committed:
Update linux_develop_workflow.yml

fwild committed:
Update linux_develop_workflow.yml

fwild committed:
Update linux_develop_workflow.yml

fwild committed:
Update linux_develop_workflow.yml

cawke committed:
old prefabs has been removed

cawke committed:
small fix

cawke committed:
Merge pull request #1905 from WEKIT-ECS/ashubin/cleanup

cleanup

Alvareon committed:
update the model library tab

Alvareon committed:
update library assets

fwild committed:
cleaning up nuget and normal packages

fwild committed:
Merge pull request #1907 from WEKIT-ECS/fwild/1906-DotNetWinRT-LoadLibraryExW-error

cleaning up nuget and normal packages

Alvareon committed:
add search bar

Alvareon committed:
add search functionality

Alvareon committed:
add comment

fwild committed:
assembly defs set (assets/plugins, assets/packages/dotnetwinrt), removed from packages.config

fwild committed:
Merge pull request #1910 from WEKIT-ECS/fwild/1906-DotNetWinRT-LoadLibraryExW-error

assembly defs set

fwild committed:
Update windows_develop_workflow.yml

removed 
          - hlversion: 1
            hlplatform: x86

Alvareon committed:
add scrollbar vertical

fwild committed:
Update windows_develop_workflow.yml

testing version updates for windows build process on windows dev pipeline

fwild committed:
Update windows_develop_workflow.yml

rolling back

fwild committed:
removed DotNetWinRT from nuget ./packages.config

fwild committed:
added modified Assets/packages.config

fwild committed:
restorning msc/csc just in case

fwild committed:
added Keystore/ README

fwild committed:
keystore update

fwild committed:
Merge pull request #1920 from WEKIT-ECS/fwild/1906-DotNetWinRT-LoadLibraryExW-error

removed DotNetWinRT from nuget ./packages.config

cawke committed:
open ai assistant has been added

cawke committed:
Merge pull request #1922 from WEKIT-ECS/ashubin/openai_assistant

open ai assistant

cawke committed:
Merge pull request #1923 from WEKIT-ECS/release_2.6

Release 2.6 to develop

fwild committed:
added dependencies hunter and moved UnitTests into MirageXR folder

fwild committed:
set min android to 8

fwild committed:
Merge pull request #1927 from WEKIT-ECS/fwild/android-settings-fix2

set min android to 8

Alvareon committed:
hide scrollbar background

fwild committed:
fixed model animations

Alvareon committed:
UI improvement

Alvareon committed:
Merge pull request #1917 from WEKIT-ECS/silyasov/home_screen_slider

add scrollbar vertical to the Home Screen

Alvareon committed:
increased the left margin of the text

fwild committed:
Merge pull request #1924 from WEKIT-ECS/fwild/1898-dead-code-detection

dependencies hunter

fwild committed:
changes

cawke committed:
fix for OpenAIManager.DeleteAssistantAsync(...)

cawke committed:
Merge pull request #1932 from WEKIT-ECS/release_2.6

release 2.6 to develop

fwild committed:
Update Assets/MirageXR/ContentTypes/3DModel/Model.cs

Co-authored-by: Alexander Shubin <icawke@gmail.com>

fwild committed:
Merge pull request #1929 from WEKIT-ECS/fwild/1928-model-animation-fix

fixed model animations

fwild committed:
removing dependency hunter (to add again, with our changes accepted now)

fwild committed:
re-added dependency hunter (with PR merged by AlexeyPerov)

fwild committed:
Merge pull request #1912 from WEKIT-ECS/silyasov/update_models_library

Update models library UI

fwild committed:
Merge pull request #1933 from WEKIT-ECS/fwild/1898-dead-code-detection

1898 dead code detection (updated package)

Alvareon committed:
bugfix

fwild committed:
Merge pull request #1934 from WEKIT-ECS/silyasov/asset_bugfix

small bugfix

Alvareon committed:
fix the tab switching

Alvareon committed:
delete an unnecessary field

Alvareon committed:
make textures readable

fwild committed:
Update README.md

Alvareon committed:
hide size temporarily

cawke committed:
Merge pull request #1938 from WEKIT-ECS/silyasov/model_editor_bugfix

model editor bugfix

andy-kanta committed:
Add UI - need to figure out how to connect it to the function. - work in progress.

andy-kanta committed:
Add UI - need to figure out how to connect it to the function. - work in progress.

andy-kanta committed:
Add the Task station config. -  Work in progress

andy-kanta committed:
Add the Task station config. -  Work in progress

andy-kanta committed:
Clean up some death code and add documentation.

andy-kanta committed:
Clean up some death code and add documentation.

fwild reviewed (changes requested):
I tested it, but for me the toggle has no effect? I don't see any error message either?
Also: please roll back the files from the commit that are not relevant - you changed a lot of packages, other prefabs (like ghosteditor), a PR should only contain what has been changed!

andy-kanta committed:
Rollback

andy-kanta committed:
Rollback

andy-kanta committed:
Added Visibility Manager and related functionality in MirageXR

Added new scripts: VisibilityManager, TaskStationVisibilityController, GameObjectVisibilityController, and ToggleVisibilityController, implementing functionality for controlling visibility of game objects. Additionally, changes have been made in the scene files reflecting the addition of this new feature.

andy-kanta committed:
Add VisibilityManagerPrefab to MirageXR UI

A new prefab named VisibilityManagerPrefab has been created and added to the MirageXR UI scripts under the directory /NewUI/VisibilityManager. This prefab includes a number of configurations for GameObject and Transform components.

andy-kanta committed:
Merge remote-tracking branch 'origin/1806-task-station-diamond-make-configurable' into 1806-task-station-diamond-make-configurable

# Conflicts:
#	Assets/MirageXR/Design/3DModels/PlayerTaskStation.prefab
#	Assets/MirageXR/Design/MobileUI/Prefabs/Content.prefab

andy-kanta committed:
Remove TaskStationVisibilityController and ToggleTaskStation

The TaskStationVisibilityController and ToggleTaskStation scripts from the NewUI directory in MirageXR have been deleted. These scripts were previously used to manage the visibility of a game object based on a toggle, however they are no longer needed.
